UT4MasterServer/Controllers/UnrealTournamentMatchmakingController.cs

```diff
@@ -258,8 +279,10 @@ public async Task<IActionResult> RemovePlayer(string id, [FromBody] EpicID[] pla
 
 		foreach (var player in players)
 		{
-			server.PublicPlayers.Remove(player);
-			server.PrivatePlayers.Remove(player);
+			if (!server.PublicPlayers.Remove(player))

```

Comment:
What is server.PublicPlayers.Remove supposed to return? Bool result? Or you meant Any()?

Reply:
it returns `true` if it was removed, otherwise `false`. in case player was removed from `PublicPlayers` then it shouldn't also exist in `PrivatePlayers`.